### ISSUE TITLE
The result of DenseCRF in Table 1

### ISSUE THREAD

**Issue by oooooppppsk** [chenqi1126/SIPE#8]
Hi, I try to reproduce the results of Table 1 in the paper. But the result of +DenseCRF is only 62.05 (using your trained weights to infer the cam file). Could you share your CRF process code and CRF hyper-parameters? Thanks!

**Comment by chenqi1126** [comment]
Thanks for your attention. The CRF process is ensembled in training IRN. You can run [cam2ir.py](https://github.com/chenqi1126/SIPE/blob/main/cam2ir.py) to obtain IR labels. Noted that the unconfident area is treated as background in evaluation for a fair comparison.

**Comment by oooooppppsk** [comment]
@chenqi1126 ,Thanks for your reply. So  the result of +DenseCRF is  the performance of IR Labels? And the evaluation file is [eval_sem_seg.py](https://github.com/chenqi1126/SIPE/blob/main/eval_sem_seg.py)? I am not sure whether my understanding is correct. Please correct me if I am wrong. Thank!

**Comment by chenqi1126** [comment]
@oooooppppsk, 

1. If you read the [cam2ir.py](https://github.com/chenqi1126/SIPE/blob/main/cam2ir.py) carefully, it is easy to find that the generation of IR labels is essentially a denseCRF process for the localization map, together with an auxiliary process for the background-weaken localization map that aims to determine the uncertainty area. When evaluating the IR labels, we treat the uncertainty area as the background. which equals the denseCRF process for localization map.
2. [eval_sem_seg.py](https://github.com/chenqi1126/SIPE/blob/main/eval_sem_seg.py) is ok, but as I say, you need to add a preprocess to convert the uncertainty area (255) to the background (0) before calculation.

**Comment by oooooppppsk** [comment]
@chenqi1126 Thanks for your reply!

**Comment by geek-APTX4869** [comment]
How to reproduce Table 1 and how to set argument `irn_network`？
